<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRulesViewer</class>
 <widget class="QWidget" name="FormRulesViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset theme="acroread"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Page â„–</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxCurrentPage">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPageLimit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Find text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFind"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>v</string>
       </property>
       <property name="icon">
        <iconset theme="go-down">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>^</string>
       </property>
       <property name="icon">
        <iconset theme="go-up">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPercent">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxPercent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FormRulesViewer</receiver>
   <slot>pageResize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFind</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormRulesViewer</receiver>
   <slot>textSearch(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pageResize(int)</slot>
  <slot>textSearch(QString)</slot>
 </slots>
</ui>
